src/network/ctf_client_datas_message.cpp

```diff
@@ -16,6 +16,7 @@ void ClientDatasMessageSenderSystem::Init()
 {
     MessageSenderSystem::Init();
         //This one is not used yet you should wire it before use
```

Comment:
Do I need to do anything here (and the other place) besides removing the comment?